test/core-test/time_evolution.jl

```diff
@@ -1,32 +1,48 @@
 @testset "Time Evolution and Partial Trace" verbose = true begin
+    # Global definition of the system
+    N = 10
+    a = kron(destroy(N), qeye(2))
+    σm = kron(qeye(N), sigmam())
+    σz = qeye(N) ⊗ sigmaz()
+
+    g = 0.01
+    ωc = 1
+    ωq = 0.99
+    γ = 0.1
+    nth = 0.001
+
+    # Jaynes-Cummings Hamiltonian
+    H = ωc * a' * a + ωq / 2 * σz + g * (a' * σm + a * σm')
+    ψ0 = kron(fock(N, 0), fock(2, 0))
+    tlist = range(0, 1000, 1000)
```

Comment:
Is it necessary to define `tlist` here ?

Cause I saw sometimes you define `tlist` below, sometimes not.

Or maybe just define `tlist` in each `testset` individually to avoid ambiguity?